lang_scala/parsing/AST_scala.ml

```diff
@@ -236,6 +236,7 @@ and pattern =
   | PatUnderscoreStar of tok (* '_' *) * tok (* '*' *)
 
   | PatDisj of pattern * tok (* | *) * pattern
+  | PatEllipsis of tok
```

Comment:
I usually add a (* sgrep: *) or (* semgrep: *) to indicate those constructs are relevant only
for semgrep patterns and are not regular Scala constructs.